clicker/src/App.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import './App.css';
+
+
+export default class Clicker extends React.Component{
+
+  constructor(props) {
+    super(props);
+    this.state = {count: props.count};
+
+    this.incrementCount = this.incrementCount.bind(this);
+    this.decrementCount = this.decrementCount.bind(this);
+    this.resetCount = this.resetCount.bind(this);
+    
+  }
+
+  incrementCount() {
+    this.setState((prevState) => ({ count: prevState.count + 1 }));
+  }
+
+  decrementCount() {
+    this.setState((prevState) => ({ count: prevState.count - 1 }));
+  }
+
+  resetCount() {
+    this.setState(() => ({ count: Clicker.defaultProps.count }));
+  }
+
+  render(){
+    return (
+      <div className="clicker">
+        <div id = "output"> {this.state.count} </div>
+        <button id = "plusButton" onClick={this.incrementCount}>+</button>
```

Comment:
```suggestion
        <button id="plusButton" onClick={this.incrementCount}>+</button>
```
Please, get rid of all extra spaces